lib/moon/design/loader.ex

```diff
@@ -0,0 +1,88 @@
+defmodule Moon.Design.Loader do
+  @moduledoc false
+
+  use Moon.StatelessComponent
+
+  import Moon.Helpers.MergeClass
+
+  prop(size, :string, values: ["2xs", "xs", "md", "lg"], default: "md")
+  prop(color, :string, default: "hit", values: Moon.colors())
+  prop(class, :css_class)
+
+  def render(assigns) do
+    ~F"""
+    <div class={
+      "relative",
+      get_size_class_by_size(@size)
+    }>
+      <div class={
+        merge([
+          "box-border block absolute w-full h-full border-solid rounded-[50%]",
+          "border-r-transparent border-b-transparent border-l-transparent",
+          "animate-loader",
+          @class
+        ]),
+        "border-#{@color}",
```

Comment:
these classes shoulb also inside merge function. and do not forget - @class is the latest

Reply:
If these classes are placed inside merge function, colours don't appear properly. That was the reason of placing them separate.

Reply:
yeah, u're right for some of the classes. will dsscuss tomm